feat: [SC-25948] :wrench: Update code organization in NameGraph SDK (#486)
namehash/namekit: packages/namegraph-sdk/README.md

# NameGraph SDK

> A TypeScript SDK for interacting with the NameGraph APIs, providing access to the world's largest collection of word-wiki names. This SDK enables easy integration with the NameGraph API endpoints for name and collection suggestions.

Reviewer: ```suggestion
A TypeScript SDK for interacting with the NameGraph APIs, providing access to the world's largest collection of names. This SDK enables easy integration with the NameGraph API endpoints for name and collection suggestions.
```

> A TypeScript SDK for interacting with the NameGenerator APIs, providing access to the world's largest collection of word-wiki names. This SDK enables easy integration with the NameGenerator API endpoints for name and collection suggestions.

Reviewer: ```suggestion
A TypeScript SDK for interacting with the NameGraph APIs, providing access to the world's largest collection names. This SDK enables easy integration with the NameGraph API endpoints for name and collection suggestions.
```

## Installation

```bash
npm install namegraph-sdk
# or
yarn add namegraph-sdk
```

## Quick Start

```typescript
import { createClient } from "namegraph-sdk";

// Create a client instance

> const client = createClient();

Reviewer: ```suggestion
const client = createNameGraphClient();
```

This is verbose but I think it's important to make it more explicit since there's so many different interpretations of createClient.

Suggesting to rename this everywhere.

Reviewer: @FrancoAguzzi Hey this is still an open action item. Thanks
